routers/api/v1/repo/pull.go

```diff
@@ -417,7 +417,7 @@ func CreatePullRequest(ctx *context.APIContext) {
 		}
 	}
 
-	if err := pull_service.NewPullRequest(ctx, repo, prIssue, labelIDs, []string{}, pr, assigneeIDs); err != nil {
+	if err := pull_service.NewPullRequest(ctx, repo, prIssue, labelIDs, []string{}, pr, assigneeIDs, []int64{}); err != nil {
```

Comment:
Maybe we can also add reviewers here like assigneeIDs?

Reply:
I've added the API part now.